web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -318,8 +318,10 @@ export type AwsOidcDeployServiceRequest = {
   region: Regions;
   subnetIds: string[];
   taskRoleArn: string;
-  databaseAgentMatcherLabels: Label[];
+  databaseAgentMatcherLabels?: Label[];
```

Comment:
Is this field used anywhere in the UI? Grepping the project by its name doesn't yield any references to it.